homeassistant/components/bring/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/bring",
   "integration_type": "service",
   "iot_class": "cloud_polling",
-  "requirements": ["python-bring-api==3.0.0"]
+  "requirements": ["homeassistant-bring-api==0.1.0"]
```

Comment:
Why do you name the library after Home Assistant? Isn't it built for general purpose for all users that want to use the Bring API? I'd suggest renaming it and remove the reference to Home Assistant.

Reply:
No, the previous is exactly for that (general purpose) and some features we want to use are not compatible.